apps/frontend/src/app/5_pages/StakePage/components/VestingStakesFrame/hooks/useGetVestingStakes.ts

```diff
@@ -0,0 +1,186 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { constants } from 'ethers';
+
+import { useAccount } from '../../../../../../hooks/useAccount';
+import { useGetProtocolContract } from '../../../../../../hooks/useGetContract';
+import { getRskChainId } from '../../../../../../utils/chain';
+import { Vesting, VestingData } from '../VestingStakesFrame.types';
+
+export const useGetVestingStakes = () => {
```

Comment:
This is unnecessary because we have `VestingContract` entity on the subgraph, take a look at my [PR (Vesting rewards)](https://github.com/DistributedCollective/sovryn-dapp/pull/436) where I fetch vesting contracts from the subgraph directly (consult with @creed-victor if something's not clear but you should do it the same way as I do it in my PR).